internal/exec/stages/disks/disks.go

```diff
@@ -437,3 +438,21 @@ func (s stage) readFilesystemInfo(fs types.Mount) (filesystemInfo, error) {
 
 	return res, err
 }
+
+// Do the minimal amount of canonicalization required to make two valid
```

Comment:
NIT: Function comments are supposed to start with the function name so Rob Pike doesn't need to use `-B` with grep.